modules/formulize/templates/admin/element_optionlist.html

```diff
@@ -54,5 +63,65 @@
     $("[name=cleardef]").blur();
   });
   <{/if}>
+  
+  <{if $content.type eq "select"}>
+  checkNAMES();
+  /**
+   *this function will consantly check for {USERNAMES} and {FULLNAMES}
+   */
+  function checkNAMES(){
+    var x=0,y=-1,z=-1;    //y for {FULLNAMES}  z for {USERNAMES}
+    for(x=0;x<length;x++){

```

Comment:
what i was doing here is to check all the options. Like they have {USERBANES} in first box, then they have "asd" in second box, and there is another {FULLNAMES} in third box.  i need to loop through all the possible boxes to make sure either {FULLNAMES} and/or  {USERNANES}  are/is checked. 

Do you want me to make sure if they got a  {USERNANES}  or {FULLNAMES} then everything else is discarded?
Or if they got  {USERNANES}  or {FULLNAMES} selected, we automatically uncheck everything else but keep the rest?
To me, make sure one of those as first option seems unreasonable, they might have other options but not checked.
But, {USERBANES}  or {FULLNAMES} is only option checked  sounds good.